src/main/java/com/adobe/cq/testing/junit/rules/Page.java

```diff
@@ -67,8 +70,16 @@ protected String initialValue() {
     };
 
     public Page(Instance quickstartRule) {
+        this((Callable<CQClient>) () -> quickstartRule.getAdminClient().adaptTo(CQClient.class));
+    }
+
+    public Page(UserRule userRule) {
+        this((Callable<CQClient>) () -> userRule.getClient());
+    }
+
+    public Page(Callable<CQClient> clientCallable) {
```

Comment:
I'd go fancier and use `Supplier` :smile: 
```suggestion
    public Page(Supplier<CQClient> clientSupplier) {
```